crates/re_types/src/blueprint/components/space_view_class_ext.rs

```diff
@@ -0,0 +1,8 @@
+use super::SpaceViewClass;
+
+impl From<&str> for SpaceViewClass {
```

Comment:
weird to only have the from &str and not from String like with name_ext. For better solution see `name_ext.rs` comment